mysite/announcements/models.py

```diff
@@ -32,3 +31,24 @@ def clean(self):
             raise ValidationError("The date and time can't be in past")
         if self.date_time_expire <= self.date_time_to_publish:
             raise ValidationError("Scheduled time can't be less than expiry time")
+
+
+STATUS_CHOICES = (
+    ('seen', 'SEEN'),
+    ('yet to receive', 'YET TO RECEIVE'),
+    ('expired', 'EXPIRED'),
+    ('user left group', 'USER LEFT GROUP')
```

Comment:
It is best to use integer values whenever we have some constants values to deal with. 

Why?
- Takes less space.
- Fewer chances of mistakes because if you use string, you get in so much mess like spelling/grammatical and other mistakes that can happen.